.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,52 +1,67 @@
-## Pull Request (PR) Checklist
+## Pull request (PR) checklist
+
 Please check if your pull request fulfills the following requirements:
+
+<!-- 💡 Tip: Tick checkboxes like this: [✅]. -->
+
 - [ ] The PR is submitted to the `main` branch.
 - [ ] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) and my PR doesn't break the rules.
 - [ ] I've read and understand the [Developer Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Guidelines.md).
 - [ ] I confirm that I've run the code locally and everything works as expected.
-- [ ] 🎬 I've attached a **screen recoding** of the changes.
 - [ ] My PR includes only the necessary changes to fix the issue (i.e., no unnecessary files or lines of code are changed).
+- [ ] 🎬 I've attached a **screen recording** of using the new code to the next paragraph.
+
+
+## Your screen recording of the changes ❗IN USE❗:
```

Comment:
I don't understand the "IN USE" part. Also you can mention that the screen recording can be skipped if not applicable